vulcanai/net.py

```diff
@@ -21,19 +22,16 @@
 
 from ops import activations, optimizers
 
-import json
-
-import cPickle as pickle
-
 from sklearn.utils import shuffle
-
+import matplotlib.pyplot as plt
 import matplotlib
+
 if os.name is not "posix":
     if "DISPLAY" not in os.environ:
         matplotlib.use('Agg')
 else:
     matplotlib.use('TkAgg')
-import matplotlib.pyplot as plt
+
```

Comment:
This should return an error when changing the backend after importing matplotlib.plt. It's the reason why it was specified after the .use()

Reply:
I tested the code and the methods were not interrupting so I didn't get any error on my side

Reply:
Ok interesting. Sounds good then!